app/javascript/components/auth-key-pair-cloud/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+
+import MiqFormRenderer from '../../forms/data-driven-form';
+import createSchema from './auth-key-pair-cloud-form.schema';
+import miqRedirectBack from '../../helpers/miq-redirect-back';
+import { API } from '../../http_api';
+
+
+const AuthKeypairCloudForm = () => {
+  const [fields] = useState([]);
+  //   const cancelUrl = `/auth_key_pair_cloud/create/new?button=cancel`;
+  //   const SaveUrl = `/auth_key_pair_cloud/create/new?button=save`;
+
+  const onSubmit = (values) => {
+    miqSparkleOn();
+    const request = API.post('/api/auth_key_pairs', values);
+    request.then(() => {
+      const message = sprintf(
+
+        __('Add of Key Pair "%s" has been successfully queued.'),
+        values.name,
+      );
+      miqRedirectBack(message, undefined, 'auth_key_pair_cloud/show_list');
+    }).catch(miqSparkleOff);
+  };
+
+  const onCancel = () => {
+    const message = sprintf(
+
+      __('Add of new Key Pair was cancelled by the user'),
+    );
+    miqRedirectBack(message, 'warning', 'auth_key_pair_cloud/show_list');
+  };
+
+
+  return (
+    <MiqFormRenderer
+      schema={createSchema(fields)}
```

Comment:
Just `createSchema()` I guess, @DavidResende0's example was a little bit more complicated with dynamically loaded fields, but you don't need the argument and the state here.